community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/RelationshipTypeCheckerStep.java

```diff
@@ -116,14 +133,19 @@ protected void done()
     public Object[] getRelationshipTypes( long belowOrEqualToThreshold )
     {
         List<Object> result = new ArrayList<>();
-        for ( Map.Entry<Object,AtomicLong> candidate : sortedTypes )
+        for ( Map.Entry<Object,MutableLong> candidate : sortedTypes )
         {
-            if ( candidate.getValue().get() <= belowOrEqualToThreshold )
+            if ( candidate.getValue().value <= belowOrEqualToThreshold )
             {
                 result.add( candidate.getKey() );
             }
         }
 
         return result.toArray();
     }
+
+    private static class MutableLong
```

Comment:
Can we use `org.apache.commons.lang3.mutable.*`? They should be in kernel module classpath already.